### PR TITLE
Initial Stable Release

### PR DESCRIPTION
**Pull request by Slothjams**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "cerner-one-style-icons",
+  "version": “1.0.0”,
```
Comment (Matt-Butler) [inline package.json]: Can you please update the accented quotes to normal quotes?

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
-# cerner-one-style-icons
+One Cerner Style Icons are the official icons from Cerner for use in Healthcare IT solutions. These icons are not intended to be consumed directly, but through its companion repo of Terra here  https://github.com/cerner/terra-icon
```
Comment (Matt-Butler) [inline README.md]: We updated this url: 
```
https://github.com/cerner/terra-icon
```
to
```
https://github.com/cerner/terra-core
```

**Comment by Matt-Butler** [comment]
+1

File: README.md [hunk 3]
```diff
@@ -1 +1,12 @@
-# cerner-one-style-icons
+Cerner One Style Icons are the official icons from Cerner for use in Healthcare IT solutions. These icons are not intended to be consumed directly, but through its companion repo of Terra here  https://github.com/cerner/terra-core
```
Comment (sethbroox) [inline README.md]: Change to "One Cerner Style"

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Cerner One Style Icons Changelog
```
Comment (sethbroox) [inline CHANGELOG.md]: Change to "One Cerner Style"

File: package.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "cerner-one-style-icons",
```
Comment (sethbroox) [inline package.json]: Change to "One Cerner Style"

File: package.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+  "version": "1.0.0",
+  "description": "Cerner One Style Icon Repository",
```
Comment (sethbroox) [inline package.json]: Change to "One Cerner Style"

File: package.json [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/cerner-one-style-icons.git"
```
Comment (sethbroox) [inline package.json]: Change to "One Cerner Style"

**Review by sethbroox — APPROVED** [review]
+1

**Pull request merged**